easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.3-2023b.yml

```diff
@@ -7,3 +7,4 @@ easyconfigs:
       options:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/21430
         from-commit: 8b509882d03402e2998ff9b22c154a6957e36d6b
+  - patchelf-0.18.0-GCCcore-13.2.0.eb:
```

Comment:
Please use EasyBuild v4.9.4 instead, and also pull in the updated `patchelf` from https://github.com/easybuilders/easybuild-easyconfigs/pull/21674

Reply:
no worries, this is just for proving the `EESSI-install-software.sh` still works, it's just a test build. I'll strip it out before marking this ready for review, as I don't want it to be deployed ;)